src/drivers/nuttx/lv_nuttx_image_cache.c

```diff
@@ -30,6 +30,9 @@ typedef struct {
     uint8_t * mem;
     uint32_t mem_size;
 
+    char name[sizeof(HEAP_NAME) + 10]; /**< UINT8_MAX is a maximum of 3 bits long in decimal notation,
+                                            and with the [] on both sides, 10 bytes must be enough */
```

Comment:
I don't understand this comment :sweat_smile: 

Note that `sizeof("GImageCache")` return 4 on 32 bit systems. If this the intention `sizeof(const char *)` would be cleaner.

Reply:
in fact, sizeof("GImageCache") is 12, a constant value of the string length with `\0` 🥹

Reply:
The comment would be easier to understand say: 
"+10 characters to store task pid."

Reply:
Done

Reply:
> in fact, sizeof("GImageCache") is 12, a constant value of the string length with \0 🥹

Wow, black magic :smile: But you are right!